src/apps/frontend/theme.ts

```diff
@@ -0,0 +1,23 @@
+export const primitives = {
+  accent: '#0d6efd',
+  accent600: '#0b5ed7',
+  mono100: '#495057',
+  white: '#dee2e6',
+  mono300: '#2B3035',
+  mono400: '#212529',
+  negative: 'red',
+};
+
+export const overrides = {
+  colors: {
+    buttonPrimaryFill: primitives.accent,
+    buttonPrimaryHover: primitives.accent600,
+    backgroundPrimary: primitives.mono300,
+    fontColor: primitives.white,
+    inputBackground: primitives.mono400,
+    inputPlaceholder: primitives.white,
+  },
+  border: {
+    borderColor: primitives.mono100,
+  },
+};
```

Comment:
For now, can we not create any overrides, and use whatever comes from basweb?

Reply:
Removed the primitives and overrides, and used default styling.